include/TrustWalletCore/TWEthereumChainID.h

```diff
@@ -20,6 +20,7 @@ enum TWEthereumChainID {
     TWEthereumChainIDVeChain = 74,
     TWEthereumChainIDThunderToken = 18,
     TWEthereumChainIDTomoChain = 88,
+    TWEthereumChainIDDai = 100,
```

Comment:
where does this number come from? xdai is 800 ID on slip 44: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
This number comes from the original issue, where the information was given that xdai was chainId 100. You can find the same information here: https://forum.poa.network/t/xdai-chain-resources/1769

Reply:
@vikmeup it's chain id not coin type